docs/core/unmanaged-api/metadata/interfaces/imetadatadispenserex-getcorsystemdirectory-method.md

```diff
@@ -0,0 +1,53 @@
+---
+description: "Learn more about: IMetaDataDispenserEx::GetCORSystemDirectory Method"
+title: "IMetaDataDispenserEx::GetCORSystemDirectory Method"
+ms.date: "03/30/2017"
+api_name:
+  - "IMetaDataDispenserEx.GetCORSystemDirectory"
+api_location:
+  - "mscoree.dll"
+api_type:
+  - "COM"
+f1_keywords:
+  - "IMetaDataDispenserEx::GetCORSystemDirectory"
+  - "GetCORSystemDirectory method [.NET Framework metadata]"
+topic_type:
+  - "apiref"
+---
+# IMetaDataDispenserEx::GetCORSystemDirectory Method
+
+Gets the directory that holds the current common language runtime (CLR). This method is supported only for use by out-of-process debuggers. If called from another component, it will return E_NOTIMPL.
```

Comment:
```suggestion
Gets the directory that holds the current common language runtime (CLR). This method is supported only for use by .NET Framework out-of-process debuggers. If called from another component, it will return E_NOTIMPL.
```